grafana/kvdb/column-value-usage.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": 1,
+  "gridPos": {
+    "h": 8,
+    "w": 12,
+    "x": 0,
+    "y": 0
+  },
+  "type": "timeseries",
+  "title": "[${chain}] ${database} per column value usage",
+  "description": "This panel shows the usage of each column value in the database.",
```

Comment:
I think think this can be better phrased so that it can be read as how much space all the values stored in a given column use.

Reply:
It is strange that they don't work since they are copied out from the previously working panel and copied into the repository. It could be that the `jq` formatting impacts how it works, but I would doubt it. Perhaps Grafana shows them differently when you expect the JSON panel relative to when you go directly to the JSON model in its entirety.

Reply:
The jq formatting is fine. The import succedes, but I get something unusable. I have to edit the queries and everything since the variables and their associated queries are gone. The only thing that should come from the user is the data source and the variable options should then be recomputed based on the queries.

Reply:
A picture is worth 1000 words. The variables are at the top of the screenshot: `source`, `chain`, `database`
<img width="1709" alt="Screenshot 2022-10-21 at 16 39 01" src="https://user-images.githubusercontent.com/54316454/197209525-34ae16d0-d547-4d42-a77e-83e518198aae.png">

Reply:
I can see the source, but I cannot see any of the chains, or the database options on the dashboard on my side. I am assuming these are defined in Thanos since that data source is there.

The query for chain also seems to be a bit misleading since it is defined as:

```
introspector_kvdb_values_size{}
```

Where can I find details on how to query the Thanos data source?

Reply:
Thanos is a Peometheus data source.

Reply:
There is a regexp IIRC that filters out the chain label value from that query.

Reply:
Let me rephrase the above - Do we have any documentation explaining which queries we support? To me, the panels are not visible, so I cannot see what the correct queries are to make them appear as in your screenshot.

Reply:
https://grafana.parity-mgmt.parity.io/goto/b45roRNVz?orgId=1 - this is the original dashboard that you generated the JSONs from initially and it works for me, but it seems that the introspector deployment has been down for the past day, so you should select a timeframe before that in order for the queries to return data.

Reply:
> Let me rephrase the above - Do we have any documentation explaining which queries we support? To me, the panels are not visible, so I cannot see what the correct queries are to make them appear as in your screenshot.

Since this is a Prometheus data source, all Prometheus queries should work.

Reply:
I managed to find the embedded queries in the link you shared above. Made sure to add these into the JSON model now. I changed the title of the dashboard to `KVDB Metrics` as well - it seemed more suitable.